backend/urls.py

```diff
@@ -67,15 +67,27 @@
     
 
 
-    path('author/<path:authorId1>/friends/<path:authorId2>',FriendViewSet.as_view(({
+    path('author/<path:authorId1>/friends/<path:authorId2>', FriendViewSet.as_view(({
         "get": "check_friends"
     }))),
-    path('friend/accept/',FriendViewSet.as_view(({
+    path('friend/accept', FriendViewSet.as_view(({
         "post": "post_friendship"
     }))),
-  
-    path('friendrequest/', FriendRequestViewSet.as_view(), name='friendrequest'),
 
+    path('friend/unfriend', FriendViewSet.as_view(({
+        "post": "unfriend"
+    }))),
+
+    path('author/<path:authorId>/friend', FriendViewSet.as_view(({
```

Comment:
make sure all the urls ends with /, so that our API will work for both with or without trailing slash, you may need to update the test after updating the urls